applications/java/test/javaPerftest.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+echo "This script is performing the Performance test for Java application"
+
+echo "Based on the input parameter, color marked will be deemded as functionality failure"
+echo "Please enter the Java application ingress url"
+echo "The number of arguments is: $#"
+a=$#
+if [ $a -ne 1 ]
+then
+ 
+echo "One parameter is expected"
+exit 1
+
+fi
+touch /tmp/timespent.out
+>/tmp/timespent.out
+INGRESSURL=$1
+rgs=()
+for i in {1..10};
```

Comment:
@YogeshBhatiaAWS please use one of the test tools that I mentioned to you before, we don't need to use crude methods like this. 
https://amzn-aws.slack.com/archives/C05H3674KGF/p1697495134693949

You can find other load testing tools here: https://amzn-aws.slack.com/archives/C05H3674KGF/p1697809239773699?thread_ts=1697775325.990349&cid=C05H3674KGF 

if you don't want ab you can siege, vegeta, gatling and many others. These tools produce reports per invocation, with latency distributions, etc.